projects/angular/src/data/datagrid/_datagrid.clarity.scss

```diff
@@ -514,6 +514,11 @@
         // compensate for the default checkbox margin, defined in its ::before section
         margin-top: -1 * $clr_baselineRem_0_25;
       }
+
+      &,
+      input {
+        cursor: pointer;
```

Comment:
What's the reason for this?

Reply:
It changes the cursor to pointer when the user enters the `.datagrid-select` element to indicate that the entire selection cell is clickable.

I had to put this style declaration for `.datagrid-select input` as well because without it, the cursor change back to the default cursor around the edges of the input for some reason.